docs/source/user_guide_kg_builder.rst

```diff
@@ -0,0 +1,412 @@
+.. _user-guide-kg-builder:
+
+User Guide: Knowledge Graph Builder
+########################################
+
+
+This page provides information about how to create a Knowledge Graph from
+unstructured data.
+
+.. warning::
+
+    This feature is still experimental. API changes and bug fixes are expected.
+
+    It is not recommended to use it in production yet.
+
+
+******************************
+Pipeline structure
+******************************
+
+A Knowledge Graph (KG) construction pipeline requires a few components:
+
+- A document **parser**: extract text from files (PDFs, ...)
+- A document **chunker**: split the text into smaller pieces of text, manageable by the LLM context window (token limit).
+- A chunk **embeder** (optional): compute and store the chunk embeddings
+- A **schema builder**: provide a schema to ground the LLM extracted entities and relations and obtain an easily navigable KG.
+- An **entity and relation extractor**: extract relevant entities and relations from the text.
+- A **Knowledge Graph writer**: write the identified entities and relations to a Neo4j database.
```

Comment:
Maybe it's just me being very nitpicky, but I think this section is more readable if each sentence doesn't start with "A..." or "An...". I felt the same way about the other user guides